ui/src/data/components/DataItem.jsx

```diff
@@ -51,15 +51,15 @@ function DataItem({ metadata, page }) {
         }
         rightActions={
           <>
-          {citationCount ? (
+          {citationCount != null && (
               <IncomingLiteratureReferencesLinkAction
                 itemCount={citationCount}
                 referenceType="citation"
                 linkQuery={getReferencingPapersQueryString(recordId)}
                 trackerEventId="Citations link"
                 eventCategory="Data search"
               />
-            ) : <></>
```

Comment:
No need to remove this. We can remove the upper one, line 53 and 64. So if there is a citationCount it render otherwise it passes the fragment